src/components/Article_Card.jsx

```diff
@@ -0,0 +1,41 @@
+import '../styles/ArticleCard.css'
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+const Article = () => {
+    const { article_id } = useParams();
+    const [article, setArticle] = useState(null);
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
```

Comment:
Don't forget to set isLoading to true at the beginning of the useEffect, in case an error occurs and requires re-rendering